index.html

```diff
@@ -56,5 +56,141 @@
             </div>
         </section>
     </header>
+    <section class="works" id="portfolio">
```

Comment:
- [ ] Good work you have done on your styling 👍 . I can see that there is so much margin-top and bottom for this section. I think you should consider adjusting the margins for this section to match the margin in the [Figma design template.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) 

| Figma Design | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/176786310-39f0dfe3-9b78-4c4c-ab37-a34ad3de6cbf.png)  | ![image](https://user-images.githubusercontent.com/63473584/176786400-f75d7a6e-b27a-489e-9b07-a6c03bb50580.png)  |